packages/fakta-opptjening-oms/src/components/timeline/OpptjeningTimeLine.tsx

```diff
@@ -89,8 +89,8 @@ const options = (opptjeningFomDato, opptjeningTomDato) => ({
   end: moment(opptjeningTomDato).add(1, 'months').endOf('month'),
   locale: moment.locale('nb'),
   margin: { item: 10 },
-  max: moment(opptjeningTomDato).endOf('month'),
-  min: moment(opptjeningFomDato).startOf('month'),
+  max: moment(opptjeningTomDato).add(1, 'week').endOf('week'),
```

Comment:
Skjønner ikke helt denne 🤔  det er vel ganske stor forskjell på å si start/slutt på måned og å legge på en uke i begge retninger på perioden. Har sikkert en logisk forklaring, men hva er den? 😛

Reply:
det er hvor stor tidslinja skal være. Før viste den hele måneden til oppstartsdatoen for opptjeningsperioden og hele måneden sluttdatoen ligger i. Nå viser den opptjeningsperioden samt uka før og etter. Det gjør skalaen i tidslinja mindre, men dekker likevel hele opptjeningsperioden